src/Float16.h

```diff
@@ -71,7 +58,9 @@ struct float16_t {
 #ifdef HALIDE_CPP_COMPILER_HAS_FLOAT16
     /** Cast to compiler's built-in _Float16 type. */
     explicit operator _Float16() const {
-        return *(const _Float16 *)&data;
+        _Float16 result;
```

Comment:
Just wonder why this change is needed. The old logic doesn't work?

Reply:
The old logic was technically a violation of strict aliasing rules; in practice, ~all C++ compilers would work fine with it, but one of ours complained, and using memcpy() is definitely safe (and likely to be optimized into the same code).